functional/debug_console/run.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+SCRIPT_PATH=$(dirname "$0")
+
+source "${SCRIPT_PATH}/../../lib/common.bash"
+
+extract_kata_env
+if [ -z "${INITRD_PATH}" ]; then
+	info "Skip: initrd path is empty"
+	exit 0
+fi
+
+
+info "Enable debug console"
+sudo cp "${RUNTIME_CONFIG_PATH}" "${RUNTIME_CONFIG_PATH}.old"
+
+info "Disable debug"
+sudo sed -i \
+	 -e 's/^#*\(kernel_params\).*=.*$/\1 = \"agent.debug_console\"/g' "${RUNTIME_CONFIG_PATH}" \
+	 -e 's/^\(enable_debug.*\)/#\1/g' \
+	 "${RUNTIME_CONFIG_PATH}"
+
+info "Running container"
+cont_name=debugconsole
+docker run --net=none --name=${cont_name} --runtime=${RUNTIME} -dti busybox sh
+
+console_path="$(ps aux | grep "${HYPERVISOR_PATH}" | grep -E -o "/run/vc/vm/[a-z0-9]+/console.sock")"
+tmp_file="$(mktemp)"
+expected_output="helloworld"
+command="echo ${expected_output}"
+
+info "Connect socat to ${console_path}"
+# run socat to allocate a new pts
+{ sudo socat -d -d unix-client:"${console_path}" pty,echo=0,raw & } > "${tmp_file}" 2>&1
+# wait for socat
+sleep 5
```

Comment:
It would be good if we could avoid sleeping and for this amount of time if possible ;)